packages/prettier-config/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Should we have prettier as a peerDependency? I know this package would do nothing if prettier is not installed, but at least we can specify a prettier version (e.g.: `<2.0.0` or `^2.0.0`)